packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -201,6 +226,15 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
         setUser,
       },
       services: {
+        async signInService(context, event) {
+          const { user, authAttributes, formValues } = context;
+
+          const username =
+            get(user, 'username') || get(authAttributes, 'username');
+          const password = get(formValues, 'password');
```

Comment:
I grabbed the password from formValues, I did not see it in authAttributes...

Reply:
yeah, this is on me and I should clean this up 😆  I'll follow up with a pr that clarifies `authAttributes` vs `formValues` after GA.